Update 'fdmprinter.def.json'

GregValiant opened the pull request:
This will be a draft.  The changes to the tooltips should be OK, but the others I'm not 100% sure on.

Alter two settings:
'material_initial_print_temperature' and 'material_final_print_temperature'
Change the tooltips to better describe the settings (the current wording is not clear), and change the "enabled" logic to include " and machine_extruder_count > 1".  (That will need to be checked but it doesn't appear that the settings are used for single extruder prints.)
Tooltip changes will of course require translation changes.
Changed the "type" from float to int as most machines aren't really set up for 1/100 of a degree.
Changed the minimum_value from "-273.15" to "0" because it was just silly.
Changed the maximum_value 
Changed the default value from 200 to material_print_temperature 

This fixes... OR This improves... -->

This is in response to a couple of bug reports including #20168 and #19190 and maybe #14019 

## Type of change

<!-- Please delete options that are not relevant. -->

- [ X] Improvement / clarification

# How Has This Been Tested?

I have had both settings hidden for years.

**Test Configuration**:
* Operating System:
Win10Pro and Cura versions since 4.8 (although the settings were disabled using definition file overrides).

# Checklist:
<!-- Check if relevant -->

- [X ] My code follows the style guidelines of this project as described in [UltiMaker Meta](https://github.com/Ultimaker/Meta) and [Cura QML best practices](https://github.com/Ultimaker/Cura/wiki/QML-Best-Practices)
- [ X] I have read the [Contribution guide](https://github.com/Ultimaker/Cura/blob/main/CONTRIBUTING.md) 
- [ not applicable ] I have commented my code, particularly in hard-to-understand areas
- [ X] I have uploaded any files required to test this change

HellAholic commented:
Tooltip change and number of extruders check is good. The setting is for multi-extruder printers mainly for when a switch happens.

I would not change the other settings as there could be side effects.
So I've heard so many stories about people printing all kinds of stuff on FDM printers, ranging from metal to chocolate to ceramics to ice-cream :) [Why are you against ice-cream printers :)] So while the range for average user might not be useful it allows for creativity and experimentation. Same with the float vs int.
That's just my personal 2 cents about this.

GregValiant commented:
I reverted everything except for the tooltips and the "enabled" logic.

HellAholic commented:
I think -190 is the sweet spot xD with liquid nitrogen but I think the original implementation was like: "If you wanna do something wrong, do it right" and went for the easy sure way. (these are just speculations, any similarities between this and any character, real or fictional is completely accidental).

GregValiant commented:
This last commit is in response to a bug report ( #20466 ) that was caused because there was no "minimum value" called out for "Minimum Even Wall Line Width" and "Minimum Odd Wall Line Width".  I added:
"minimum_value": "0.4 * machine_nozzle_size" to both settings.

"Z-seam position" dropdown:  Re-ordered the settings so they go clockwise from the left front. 
Added a "Middle" setting and adjusted the Zseam X and Y boxes to show the bed midpoint when "Middle" is selected.

GregValiant commented:
Commit 6 is a bug fix for #20499 .  Changing the "coasting_volume" {"minimum_value"} for "0.0" to "0.001" fixed the problem although the real problem is likely somewhere else in the code.
